2021-07-19 04:31:09,310-[INFO]-my_log.py-api_log-日志信息:-----正在测试login模块里面第3条测试用例：输入错误的手机号,
2021-07-19 04:31:09,312-[INFO]-my_log.py-api_log-日志信息:测试数据是：{'mobilephone': '18300070759', 'pwd': '123456'}
2021-07-19 04:31:09,426-[INFO]-my_log.py-api_log-日志信息:实际结果是:{'status': 0, 'code': '20111', 'data': None, 'msg': '用户名或密码错误'}
2021-07-19 04:31:09,428-[INFO]-my_log.py-api_log-日志信息:一条测试用例执行完毕
